lambdas/tests/unit/handlers/test_edge_presign_handler.py

```diff
@@ -1,21 +1,22 @@
+import copy
+import json
 from unittest.mock import Mock
 
 import pytest
-from botocore.exceptions import ClientError
-from services.edge_presign_service import EdgePresignService
+from handlers.edge_presign_handler import lambda_handler
 from tests.unit.enums.test_edge_presign_values import (
-    ENV,
-    EXPECTED_DYNAMO_DB_CONDITION_EXPRESSION,
-    EXPECTED_DYNAMO_DB_EXPRESSION_ATTRIBUTE_VALUES,
-    EXPECTED_EDGE_NO_CLIENT_ERROR_CODE,
-    EXPECTED_EDGE_NO_CLIENT_ERROR_MESSAGE,
-    EXPECTED_SSM_PARAMETER_KEY,
-    MISSING_ORIGIN_EVENT_MODEL,
-    NHS_DOMAIN,
+    EXPECTED_DOMAIN,
+    EXPECTED_EDGE_MALFORMED_ERROR_CODE,
+    EXPECTED_EDGE_MALFORMED_ERROR_MESSAGE,
+    EXPECTED_EDGE_MALFORMED_HEADER_ERROR_CODE,
+    EXPECTED_EDGE_MALFORMED_HEADER_ERROR_MESSAGE,
+    EXPECTED_EDGE_MALFORMED_QUERY_ERROR_CODE,
+    EXPECTED_EDGE_MALFORMED_QUERY_ERROR_MESSAGE,
+    EXPECTED_EDGE_NO_ORIGIN_ERROR_CODE,
+    EXPECTED_EDGE_NO_ORIGIN_ERROR_MESSAGE,
     TABLE_NAME,
     VALID_EVENT_MODEL,
 )
-from utils.lambda_exceptions import CloudFrontEdgeException
 
 
 def mock_context():
```

Comment:
May have missed this the first time round reviews, but there is already a mocked context fixture in `tests/unit/conftest.py` could we remove this and use the already defined one please?